content/blog/what’s-new-with-chapel-nine-questions-for-the-development-team.md

```diff
@@ -0,0 +1,17 @@
+---
+title: What’s New with Chapel? Nine Questions for the Development Team
+date: 2024-09-04T16:54:10.515Z
+featuredBlog: true
+externalLink: https://www.hpcwire.com/2024/09/04/whats-new-with-chapel-nine-questions-for-the-development-team/
+author: HPE DEV
```

Comment:
Hi Brad,
As for your question for the author of the interview, you can keep the HPE DEV and image (avatar1.svg) or you may want to change the authorimage to be the Chapel image if you prefer. 
You can also put your name as the Author of the post if you have conducted the interview.
Up to you. 

As for the feature blog page, I will change the featured blog as per your request:

I will remove the post "Chapel 2.0: Scalable and Productive Computing for All" as feature blog post, and position your new blog post before the "Reflections on ChapelCon '24: A Community Growing Together".
Is it what you wish?
Or would you want me to move your latest blog post (What’s New with Chapel? Nine Questions for the Development Team) higher up in the featured blog section? 

Could you please confirm?
Thanks
Denis

Reply:
Hi Denis — 

Thanks for taking a quick look!  With respect to:

> As for your question for the author of the interview, you can keep the HPE DEV and image (avatar1.svg) or you may want to change the authorimage to be the Chapel image if you prefer.
You can also put your name as the Author of the post if you have conducted the interview.
Up to you.

I'd be inclined to leave it as-is for simplicity (or maybe "out of laziness" :D ), but if you have a strong preference to update either the image or author name, I'm game to do that as well.  For reference, I did write the responses to the interview questions and Doug Eadline wrote the questions and the introductory text for the interview.

> I will remove the post "Chapel 2.0: Scalable and Productive Computing for All" as feature blog post, and position your new blog post before the "Reflections on ChapelCon '24: A Community Growing Together".
Is it what you wish?
Or would you want me to move your latest blog post (What’s New with Chapel? Nine Questions for the Development Team) higher up in the featured blog section?

Yes, please remove the "Chapel 2.0…" card and add this one as a featured card.  As far as ordering goes, I'm not sure what determines the current ordering (e.g., whether it's by date, or how "featured" an article is, or some combination of the two) but in any case I'd like this one to go earlier / to the left of "Reflections on ChapelCon '24…".  I'll leave it to you to decide how to position it with respect to the other non-Chapel cards if that's OK, since you're more familiar with them and their importance/impact?

Thanks again!